build.assets/Makefile

```diff
@@ -144,16 +144,15 @@ buildbox-centos6:
 
 #
 # Builds a Docker buildbox for CentOS 6 FIPS builds
+# Teleport FIPS builds fail on CentOS 6 builds as of Go 1.16, so have been disabled for Teleport 7.0+
+# This is due to the lower glibc version used with CentOS 6 not supporting the 'getauxval' syscall,
+# preventing Go's boringcrypto fork being compiled from source.
+# https://github.com/gravitational/teleport/issues/7207
 #
 .PHONY:buildbox-centos6-fips
 buildbox-centos6-fips:
```

Comment:
should this rule just get removed entirely?

Reply:
I suppose as nobody external builds this code and we're not intending to add it back, we can just remove all the CentOS 6 FIPS targets completely.